common/amathutils_lib/package.xml

```diff
@@ -13,7 +13,6 @@
   <buildtool_depend condition="$ROS_VERSION == 2">ament_cmake_auto</buildtool_depend>
   
   <build_depend>carma_cmake_common</build_depend>
-  <build_depend condition="$ROS_VERSION == 2">ament_auto_cmake</build_depend>
```

Comment:
Why is this no longer needed?

Reply:
Looks like its already there on line 13, also there's a typo